modules/perception/camera/tools/offline/visualizer.h

```diff
@@ -59,12 +59,17 @@ class Visualizer {
       const CameraFrame &frame,
       const base::MotionBufferPtr motion_buffer,
       const Eigen::Affine3d &world2camera);
-  void Draw2Dand3D_all_info_single_camera(const cv::Mat &img,
-                                          const CameraFrame &frame,
-                                          const Eigen::Matrix3d intrinsic,
-                                          const Eigen::Matrix4d extrinsic,
-                                          const Eigen::Affine3d &world2camera);
+  void Draw2Dand3D_all_info_single_camera(
+      const cv::Mat &img,
+      const CameraFrame &frame,
+      const Eigen::Matrix3d intrinsic,
+      const Eigen::Matrix4d extrinsic,
```

Comment:
Is it possible to be `const reference`?

Reply:
Yes. I changed them to "const reference"